apps/bfd-pipeline/bfd-pipeline-app/src/main/assembly/bin/bfd-pipeline-app.sh

```diff
@@ -4,6 +4,8 @@ scriptDirectory="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 bfdPipelineAppJar="$(echo ${scriptDirectory}/bfd-pipeline-app-*.jar)"
 classpath="${bfdPipelineAppJar}:${scriptDirectory}/lib/*"
 mainClass="gov.cms.bfd.pipeline.app.PipelineApplication"
+export BFD_JAVA_HOME=/usr/local/openjdk-11/
```

Comment:
Adding this here means that this script will only work when java is in that location. I don't think we can make that assumption. Also I'm not sure why this fixes the issue since I thought all of the other tests that don't have invalid configuration were working fine so they must have found java, right?

Reply:
for these scripts it doesnt find java, which is weird..